force-app/main/default/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -488,7 +488,7 @@ export default class GeFormRenderer extends LightningElement{
     }
 
     handleNullPaymentFieldsInFormState() {
-        if (this.shouldNullPaymentRelatedFields()) {
+        if (this.shouldNullPaymentRelatedFields() || this.shouldNullFormerAchPayment()) {
```

Comment:
This change was partially a guess based on the logic I'm seeing, I just want to make sure ACH has the same guard rails in place as Credit Card!

Reply:
Don't you think we should have the logic in one function?